librz/bin/dwarf/macro.inc

```diff
@@ -0,0 +1,124 @@
+#define OK_None \
+	do { \
+		*out = NULL; \
+		return true; \
+	} while (0)
+
+#define RET_VAL_IF_FAIL(x, val) \
+	do { \
+		if (!(x)) { \
+			return (val); \
+		} \
+	} while (0)
+
+#define READ8(buf) \
+	(((buf) + 1 < buf_end) ? *((ut8 *)(buf)) : 0); \
+	(buf)++
+#define READ16(buf) \
```

Comment:
There are functions in `librz/include/rz_util/rz_buf.h` for that - e.g. `rz_buf_read_ble16_offset()`, that do the same